src/Pipeline.h

```diff
@@ -60,7 +60,8 @@ class Pipeline {
     Internal::IntrusivePtr<PipelineContents> contents;
 
     std::vector<Argument> infer_arguments(Internal::Stmt body);
-    std::vector<const void *> prepare_jit_call_arguments(Realization dst, const Target &target, const ParamMap &param_map, bool is_bounds_inference);
+    std::vector<const void *> prepare_jit_call_arguments(Realization dst, const Target &target, const ParamMap &param_map,
```

Comment:
nit: prepare_jit_call_arguments() isn't called anywhere outside of Pipeline; it could be made local to that file

Reply:
It needs to be a method on `Pipeline` because it calls other methods on `Pipeline`. We could move a bunch of stuff into `PipelineContents`, but I've never really been a fan of PIMPL'ing everything.

Reply:
Doh -- somehow I missed its method-ness. Nevermind.